.travis.yml

```diff
@@ -34,6 +34,13 @@ matrix:
         - make tester ${NO_CACHE} ${SUPPRESS_LOGS} || exit 1
         - make test ${SUPPRESS_LOGS}
 
+    - language: generic
```

Comment:
Don't think it should be another spawn on its own. It will take another machine and we are bound on resources as is.

Since systems tests are the shorter one (for now) I think we can add this as a step to that job (after the tests if you are afraid of noise created by the compilation)

Reply:
We can, but currently, this job is fast (~4 min) so this has little effect. 
The upside of adding it as another spawn is that we can add to it all the frontend unit tests

Reply:
No, We cannot add the frontend tests as this build is destructive for the frontend, this is also a reason to run it as a diff spawn. I think the FE tests should be incorporated into the "Unit Tests" job

Reply:
It is not distractive as it is inside an image...

Reply:
I mean you cannot run the FE tests (on a container build with that image) after you run this code